galeracluster/source/installmariadbsrc.rst

```diff
@@ -5,61 +5,189 @@ Source Installation
 
 MariaDB Galera Cluster is the MariaDB implementation of Galera Cluster for MySQL.  Binary installation packages are available for Debian- and RPM-based distributions of Linux.  In the event that your Linux distribution is based on a different package management system, or if it runs on a different unix-like operating system where binary installation packages are not available, such as Solaris or FreeBSD, you will need to build MariaDB Galera Cluster from source.
 
-.. note:: This tutorial omits MariaDB authentication options for brevity.
 
+.. seealso:: In the event that you built MariaDB Galera Cluster over an existing standalone instance of MariaDB, there are some additional steps that you need to take in order to update your system to the new database server.  For more information, see :doc:`migration`.
+
+
+
+
+---------------------------------
+Preparing the Server
+---------------------------------
+.. _`installmariadb-prep-server`:
+
+When building from source code, ``make`` cannot manage or install dependencies for either Galera Cluster or the build process itself.  You need to install these packages first.
+
+- For Debian-based distributions of Linux, if MariaDB is available in your repositories, you can run the following command:
+
+  .. code-block:: console
+
+     # apt-get build-dep mariadb-server
+
+- For RPM-based distributions, instead run this command:
+
+  .. code-block:: console
+
+     # yum-builddep MariaDB-server
+
+In the event that neither command works for your system or that you use a different Linux distribution or FreeBSD, the following packages are required:
+
+- **MariaDB Database Server with wsrep API**: Git, CMake, GCC and GCC-C++, Automake, Autoconf, and Bison, as well as development releases of libaio and ncurses.
+
+- **Galera Replication Plugin**: SCons, as well as development releases of Boost, Check and OpenSSL.
+
+Check with the repositories for your distribution or system for the appropriate package names to use during installation.  Bear in mind that different systems may use different names and that some may require additional packages to run.  For instance, to run CMake on Fedora you need both ``cmake`` and ``cmake-fedora``.  
+
+     
 -----------------------------------------
-Build Dependencies
+Building MariaDB Galera Cluster
 -----------------------------------------
-.. _`Build Dependencies`:
+.. _`build-galera-mariadb`:
```

Comment:
Can a simpe git clone https://github.com/mariadb/server work in this case as opposed to all those individual steps?